flake.nix

```
@@ -74,7 +74,7 @@
     importNgiProjects = {
       pkgs ? {},
       sources ? {
-        configurations = rawNixosConfigurations;
+        examples = rawNixosConfigurations;
```

Begs the question why the RHS is "configurations" then?

```suggestion
        examples = rawNixosExamples;
```